app/assets/stylesheets/application.scss

```diff
@@ -340,7 +348,7 @@ table {
 .spinner-placeholder {
   width: 16px;
   height: 16px;
-  background: image-url('spinner.gif');
+  background: url('spinner.gif');
```

Comment:
This whole css class isn't used anymore.
In the next commit it will be removed.